src/components/OSCALEditableFieldActions.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import CancelIcon from "@material-ui/icons/Cancel";
+import EditIcon from "@material-ui/icons/Edit";
+import SaveIcon from "@material-ui/icons/Save";
+import { IconButton } from "@material-ui/core";
+
+export function getElementLabel(editedField) {
+  return editedField.toString().replace(/,/g, "-");
+}
+
+function getIconButtons(props) {
```

Comment:
Any reason for this being a separate function rather than in `OSCALEditableFieldActions`?

Reply:
I did it because the `getIconsButtons` also returns based on a condition (using a ternary operator), which would mean there's a nested ternary operator return statement if we included everything in one function. When I created the component, I was initially thinking that would make it difficult to read but it doesn't look to be so now.